libraries/AP_Motors/AP_MotorsHeli_RSC.cpp

```diff
@@ -33,38 +33,38 @@ const AP_Param::GroupInfo AP_MotorsHeli_RSC::var_info[] = {
 
     // @Param: MODE
     // @DisplayName: Rotor Speed Control Mode
-    // @Description: Determines the method of rotor speed control
-    // @Values: 1:Ch8 Input, 2:SetPoint, 3:Throttle Curve, 4:Governor
+    // @Description: Selects the type of rotor speed control used to determine throttle output to the HeliRSC servo channel when motor interlock is enabled (throttle hold off). RC Passthrough sends the input from the RC Motor Interlock channel as throttle output.  External Gov SetPoint sends the RSC SetPoint parameter value as throttle output.  Throttle Curve uses the 5 point throttle curve to determine throttle output based on the collective output.  Governor is ArduCopter's built-in governor that uses the throttle curve for a feed forward throttle command to determine throttle output.
+    // @Values: 1:RC Passthrough, 2:External Gov SetPoint, 3:Throttle Curve, 4:Governor
     // @User: Standard
     AP_GROUPINFO("MODE", 2, AP_MotorsHeli_RSC, _rsc_mode, (int8_t)ROTOR_CONTROL_MODE_SPEED_PASSTHROUGH),
 
     // @Param: RAMP_TIME
-    // @DisplayName: RSC Ramp Time
-    // @Description: Time in seconds for the output to the main rotor's ESC to reach full speed
+    // @DisplayName: Throttle Ramp Time
```

Comment:
I think it's a good idea to prefix the Display name with something that identifies which feature it applies to.  So for example "RSC" is more specific than "Throttle" because throttle could be either an input or output and the user doesn't really know which is involved.

Reply:
@rmackay9 The actual param name can be as mysterious and un-understable as the programmer wants, which is what programmers and engineers really like. But the display name and description is what is presented to the end user in the GCS.

So the question is, is the software supposed to be programmer techy, or user-friendly?

End users have no clue what RSC is, what it stands for, or what it does. Those same folks do understand the term "throttle", however. This is the kind of stuff that makes supporting ArduPilot unnecessarily difficult and time-consuming.

Reply:
@ChristopherOlson, I think we're all trying to get to a name and description that is user-friendly.  I'm not arguing that the old name is better.  The question is whether "throttle" is specific enough to clarify for the user what this affects.  I think "throttle" can have at least two meanings - "input throttle" and "output throttle".

Reply:
Ok, I am not aware of what input throttle vs output throttle is. There may be some confusion due to the term "throttle" being used for collective in helicopters in the code, while "throttle" in helicopters has a different meaning and it was actually coded improperly from the start. Thrust is controlled with pitch (collective), torque and power is controlled with throttle (engine). How fast the throttle ramps during runup determines torque. If throttle ramps too fast it will over-torque the engine resulting in drivetrain damage, lag the blades and destroy the helicopter, or torque twist it on the ground. Helicopter pilots understand this principle (or if they don't, they will learn it) and using "RSC" means nothing to them.

What Bill did here was was to change the display name to something helicopter pilots understand and identify with.

Reply:
OK, well you guys are the experts so I'll leave you all to decide what the best name is.

Reply:
@ChristopherOlson  and @rmackay9  I'll probably be a little wordy in the description and take the middle ground.  Hopefully it won't be confusing.  I think with a new wiki page describing the RSC, what it does and how it controls rotor speed, the user shouldn't be confused with the display name.  That is on my todo list.